docs/source/index.rst

```diff
@@ -34,6 +34,7 @@ for each of those tasks and point you to the best tools to use for your preferre
    :maxdepth: 1
 
    community_gallery/community_gallery.rst
+   extensions
```

Comment:
It seems a bit confusing to have this section appear so far from the `extensions_tutorial`. I would suggest either adding the page to the `extensions_tutorial/extensions_tutorial_home` section or have it listed as a separate top-level element directly after that section.